kano_apps/AppData.py

```diff
@@ -105,21 +111,32 @@ def get_applications(parse_cmds=True):
                     if not is_app_installed(data):
                         data["_install"] = True
 
-                    apps.append(data)
+                    _applications[f] = data
                     if "overrides" in data:
-                        blacklist += data["overrides"]
+                        blacklist.extend(data["overrides"])
 
             if f[-8:] == ".desktop" and f[0:5] != "auto_":
```

Comment:
While not explicitly to do with your changes, would be both clearer and more performant if this were an `elif` - the file extension can't be both `.app`  and `.desktop`.